bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Verify/VerifyElection.cs

```diff
@@ -0,0 +1,1087 @@
+        var voteLimits = await VerifyVoteLimits(
+            record.Constants,
+            record.Context,
+            record.Manifest,
+            record.EncryptedBallots
+        );
+        results.Add(voteLimits);
+
+        // Verification 7 (Validation of confirmation codes)
+        var confirmationCodes = await VerifyConfirmationCodes(
+            record.Constants,
+            record.Context,
+            record.Manifest,
+            record.EncryptedBallots
+        );
+        results.Add(confirmationCodes);
+
+        // Verification 8 (Correctness of ballot aggregation)
+        var ballotAggregation = await VerifyBallotAggregation(
+            record.Constants,
+            record.Context,
+            record.Manifest,
+            record.EncryptedBallots,
+            record.EncryptedTally
+        );
+        results.Add(ballotAggregation);
+
+        // Verification 9 (Correctness of decryptions)
+        var tallyDecryptions = await VerifyTallyDecryptions(
+            record.Constants,
+            record.Context,
+            record.Manifest,
+            record.EncryptedTally,
+            record.Tally
+        );
+        results.Add(tallyDecryptions);
+
+        // Verification 10 (Validation of correct decryption of tallies)
+        var tallyMetadata = await VerifyTallyDecryptionMetadata(
+            record.Constants,
+            record.Context,
+            record.Manifest,
+            record.EncryptedTally,
+            record.Tally
+        );
+        results.Add(tallyMetadata);
+
+        // Verification 11 (Correctness of decryptions of contest data)
+        results.Add(new VerificationResult(IsValidwithKnownSpecDeviations, "Verification 11 (Correctness of decryptions of contest data): TODO: E.G. 2.0 - implement"));
+
+        // Verification 12 (Correctness of decryptions for challenged ballots)
+        var ballotDecryptions = await VerifyBallotDecryptions(
+            record.Constants,
+            record.Context,
+            record.Manifest,
+            record.EncryptedBallots
+                .Where(i => i.IsChallenged)
+                .ToList(),
+            record.ChallengedBallots
+        );
+        results.Add(ballotDecryptions);
+
+        // Verification 13 (Validation of correct decryption of challenged ballots)
+        var ballotDecryptionMetadata = await VerifyBallotDecryptionMetadata(
+            record.Constants,
+            record.Context,
+            record.Manifest,
+            record.EncryptedBallots
+                .Where(i => i.IsChallenged)
+                .ToList(),
+            record.ChallengedBallots
+        );
+        results.Add(ballotDecryptionMetadata);
+
+        // Verification 14 (Correctness of contest data decryptions for challenged ballots)
+        results.Add(new VerificationResult(IsValidwithKnownSpecDeviations, "Verification 14 (Correctness of contest data decryptions for challenged ballots): TODO: E.G. 2.0 - implement"));
```

Comment:
everywhere that there is a deviation from the spec for something that isn't implemented, you will see this language. you can flip the `IsValidwithKnownSpecDeviations` flag if you want to see them print out and explicitly invalidate a result